content/en/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation.md

```diff
@@ -28,9 +28,7 @@ The most common way to implement the APIService is to run an *extension API serv
 Extension API servers should have low latency networking to and from the kube-apiserver.
 Discovery requests are required to round-trip from the kube-apiserver in five seconds or less.
 
-If your extension API server cannot achieve that latency requirement, consider making changes that let you meet it. You can also set the
```

Comment:
I feel like shortening the content makes sense, so approving this. Maybe a member can approve/